### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by traiansf**
Everybody interested, please review. Once this is merge we will release version 3.4

### CODE REVIEW

**Comment by dwightguth** [comment]
Make sure you fully specify the change in behavior to kast and kompile in #309 and #424.

**Comment by traiansf** [comment]
@dwightguth I've added some information which could be collected from the commit messages and suggested people should take a look at the usage.  If you think more info is needed, please send me an itemized list and I'll add it there.

**Comment by dwightguth** [comment]
This is failing because Jenkins is set up to test unified builtins. It's actually okay to merge. Does anyone have any other feedback to give? I'd like to merge this sometime this morning.

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	- Improved caching of rules to speedup re-compilation if no syntax changes.
+
+
+## Krun ##
+- Changed option `--output-mode` to `--output` with short-name `-o`.
+- Improved reporting of running times in the verbose output mode (Issue #169).
+- `--search` now accepts more then one token in the input stream (Issue #159).
+- Allowing syntactic sorts in `--search` patterns (Issue #114)
+- Search graph is computed by Maude and parsed by krun only if one of
+arguments `--graph`, `--debug`, or `--debug-gui` is set (Issue #37).
```
Comment (radumereuta) [inline changelog.md]: one of 'the' arguments?

**Comment by radumereuta** [comment]
lgtm

**Comment by grosu** [comment]
Also, all the other file names are capitalized: README, INSTALL, LICENSE.  Shouldn't the name of the changelog be CHANGELOG, for uniformity?

**Comment by andreistefanescu** [comment]
Yes, we should be consistent.

Andrei

**Comment by dwightguth** [comment]
Have you incorporated everyone's suggestions? I'd like to merge this if nobody else has anything they want you to change.

**Pull request merged**